lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -80,7 +80,7 @@ def init_from_structure(self, student, course_structure, scores_client, submissi
                 student, descendant_key, course_structure, scores_client, submissions_scores, persisted_values={},
             )
         self.all_total, self.graded_total = graders.aggregate_scores(self.scores, self.display_name, self.location)
-        self._log_event(log.warning, u"init_from_structure", student)
+        self._log_event(log.info, u"init_from_structure", student)
```

Comment:
Does it make sense to also bump down a few of the `CourseGrade` ones - since the ones for each individual method are always outputted and don't output additional data?  Unless they add substantive value in monitoring, they probably aren't needed, right?  For example, from the stats you have here, we can tell that 4444 CourseGrade objects were accessed, and 430,000+ SubsectionGrade objects were created.
  
Actually, stepping back a bit, from a high-level, doesn't it make sense to have the following info (instead?):

- Each time a CourseGrade object is accessed (captured in `CourseGrade.compute_and_update`)
- In that singular CourseGrade log, add additional statistics such as 
  - How many of its SubsectionGrades were read versus created.  We can tell that from (1) the count of `_unsaved_subsection_grades` and (2) the count of subsections (can be counted in `CourseGrade.compute_and_update`
  - How many total scored blocks were accessed.  We can tell that from the count of `CourseGrade.locations_to_weighted_scores`.
- Each time a grade is updated (captured in `SGF.update`)
- For any expected (`log.warning`) and unexpected (`log.error`) exceptions.